src/main/java/io/codelex/TESTS/Basket.java

```diff
@@ -0,0 +1,38 @@
+package io.codelex.TESTS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Basket<Object> {
```

Comment:
Uzdevumā bija konkrēti minēts:
`class Object cannot be used and must use generics`